client/js/classification.js

```diff
@@ -1,3 +1,5 @@
+// noinspection BadExpressionStatementJS
```

Comment:
Is this absolutely necessary?
In general it's better to not use this type of exclusions, at least on a per file level.

I.e. we should either not exclude these warnings at all or we should exclude them through some project wide config.

Reply:
No idea what this is, but removed it